src/main/java/org/opentripplanner/middleware/spark/Main.java

```diff
@@ -194,4 +205,22 @@ public static String getConfigPropertyAsText(String name, String defaultValue) {
             return defaultValue;
         }
     }
+
+    /**
+     * @return a config value (nested fields defined by dot notation "data.use_s3_storage") as an int or the default
+     * value if the config value is not defined (null) or cannot be converted to an int.
+     */
+    public static int getConfigPropertyAsInt(String name, int defaultValue) {
+
+        int value = defaultValue;
+
+        try {
+            JsonNode node = getConfigProperty(name);
+            value = Integer.parseInt(node.asText());
+        } catch (NumberFormatException | NullPointerException e) {
+            LOG.error("Unable to parse {}. Using default: {}", name, defaultValue, e);
```

Comment:
This should be a warning since the application can proceed fine with a default value.